com.unity.netcode.gameobjects/Editor/CodeGen/NetworkBehaviourILPP.cs

```diff
@@ -20,9 +20,6 @@ namespace Unity.Netcode.Editor.CodeGen
 
     internal sealed class NetworkBehaviourILPP : ILPPInterface
     {
-        private const string k_ReadValueMethodName = nameof(FastBufferReader.ReadValueSafe);
-        private const string k_WriteValueMethodName = nameof(FastBufferWriter.WriteValueSafe);
```

Comment:
why not stick with these? I think they're better than hardcoding strings inline :(

Reply:
It was an accident. I accidentally did my changes in the wrong project, copy-pasted into testproject without realizing what I was in was out of date, and foolishly didn't look at the diff before submitting. Rollback incoming.